### PR TITLE
refactor(entity-browser): use route component instead of config

### PR DESCRIPTION
**Pull request by izzappel**
- preparation for react router upgrade
- react router v6 does only support `element` on route component
- much easier to migrate with route components

Refs: TOCDEV-4660

### CODE REVIEW

**Comment by izzappel** [comment]
Screenshot 2022-03-01 at 08 29 07
The `index.js` is without code splitting on route-level 13KB bigger.

**Comment by izzappel** [comment]
I don't think that there will be much difference in loading performance in comparison with the version on master.

File: packages/entity-browser/src/input.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  {
+    name: 'entityName',
+    action: setFormBase,
+    argsFactory: input => [input.formBase || input.entityName]
+  },
+  {
+    name: 'formBase',
+    action: setFormBase,
+    argsFactory: input => [input.formBase]
+  },
```
Comment (rzueger) [inline packages/entity-browser/src/input.js:33]: block starting on line 29 unnecessary? already handled in block on line 24? `entityName` is a mandatory prop, so the block on line 24 will always be executed an do everything that's needed for formBase, doesn't it?

**Review by rzueger — APPROVED** [review]

File: packages/entity-browser/src/input.js (hunk 1 above)
Comment (izzappel) [reply packages/entity-browser/src/input.js:33]: hm what happen if just `formBase` has changed without `entityName`?

Comment (rzueger) [reply packages/entity-browser/src/input.js:33]: yeah, you're right. 🙈  nevermind

**Pull request merged**